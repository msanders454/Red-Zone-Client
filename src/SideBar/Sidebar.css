.sidebar {
    background: linear-gradient(to right, rgb(243, 222, 222) 0%, #e9b1b1 50%, rgb(206, 27, 27) 100%);
    height: 75px;
    display: flex;
    justify-content: space-evenly;
    padding-top: 2%;
}

.sidebar a {
    text-decoration: underline;
}

li a {
    text-decoration: none;
    color: rgb(0, 0, 0);
    font-size: 13px;
}

.active {
    font-weight: bolder;
    text-decoration: underline;
}

.sidebar a {
    text-decoration: none;
}

.sidebar a:hover {
    text-decoration: underline;
}

li {
    list-style-type: none;
    color: black;
}

@media only screen and (min-width: 468px) {
    li a {
        font-size: 15px;
    }
}

@media only screen and (min-width: 618px) {
    li a {
        font-size: 16px;
    }
}

@media only screen and (min-width: 768px) {
    .sidebar {
        position: fixed;
        border: 0px solid;
        border-radius: 5px;
        width: 150px;
        height: 270px;
        display: block;
        padding: 0;
        background: linear-gradient(to right,rgb(206, 27, 27) 0%, #e9b1b1 50%, rgb(243, 222, 222) 100%);
    }
    li a {
        font-size: 13px;
    }
    li {
        margin: 30px 15px;
    }
}

@media only screen and (min-width: 968px) {
    .sidebar {
        width: 190px;
    }
    li a {
        font-size: 16px;
    }
}

@media only screen and (min-width: 1168px) {
    .sidebar {
        height: 300px;
        width: 200px;
        font-size: 17px;
    }
    li a {
        font-size: 20px;
    }
}