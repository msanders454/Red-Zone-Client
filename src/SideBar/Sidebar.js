import React from 'react';
import { NavLink } from 'react-router-dom';
import './Sidebar.css';

const SideBar = () => {
    return (
        <div className='sidebar'>

            <li>
                <NavLink to='/budget' activeClassName='active'>
                   Change Red Zone Amount
                </NavLink>
            </li>
             <li>
                <NavLink to='/expenses' activeClassName='active'>
                    All Expenses
                </NavLink>
            </li>
            <li>
                <NavLink to='/addExpense' activeClassName='active'>
                    Add Expense
                </NavLink>
            </li>
            <li>
                <NavLink to='/statistics' activeClassName='active'>
                    Am I in the RedZone?
                </NavLink>
            </li>
            
        </div>
    );
}

export default SideBar;
