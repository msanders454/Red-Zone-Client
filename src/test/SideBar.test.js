import React from "react";
import { shallow, configure } from "enzyme";
import SideBar from "../SideBar/SideBar";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });
test("should render SideBar correctly", () => {
  const wrapper = shallow(<SideBar />);
  expect(wrapper).toMatchSnapshot();
});