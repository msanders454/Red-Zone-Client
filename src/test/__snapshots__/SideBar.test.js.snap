// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SideBar correctly 1`] = `ShallowWrapper {}`;
